src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html

```diff
@@ -0,0 +1,430 @@
+        width: 100px;
+        display: flex;
+        flex-wrap: wrap;
+      }
+
+      .add-seat {
+        text-align: center;
+        width: 100px;
+        color: var(--secondary-color);
+        font-size: 32px;
+        font-weight: bold;
+        background: var(--primary-color);
+        box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
+          0 1px 5px 0 rgba(0, 0, 0, 0.12),
+          0 3px 1px -2px rgba(0, 0, 0, 0.2);
+      }
+
+      .seatmap {
+        overflow: auto;
+      }
+
+      .seatmapwrapper {
+        display: flex;
+        justify-content: space-around;
+      }
+
+      .grouplabel {
+        text-align: center;
+        width: 100%;
+        color: var(--secondary-color);
+        font-size: 32px;
+        font-weight: bold;
+        background: var(--primary-color);
+        box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14),
+          0 1px 5px 0 rgba(0, 0, 0, 0.12),
+          0 3px 1px -2px rgba(0, 0, 0, 0.2);
+      }
+
+      .non-seat-group {
+        min-width: 810px;
+        margin-top: 60px;
+        display: flex;
+      }
+
+      .non-seat-group>div {
+        background-color: var(--paper-grey-500);
+        display: inline-block;
+        text-align: center;
+      }
+
+      .non-seat-group>div>div {
+        position: relative;
+        color: #fff;
+        width: 100%;
+        top: calc(50% - 0.75em);
+      }
+
+      .non-seat-group iron-icon {
+        --iron-icon-height: 56px;
+        --iron-icon-width: 56px;
+      }
+
+      .lounge {
+        width: 300px;
+        height: 450px;
+      }
+
+      .stage {
+        width: 315px;
+        height: 180px;
+        margin-top: 270px;
+      }
+
+      .entrance {
+        width: 195px;
+        height: 330px;
+        margin-top: 120px;
+      }
+
+      #listbox {
+        overflow-y: auto;
+        max-height: 40vh;
+      }
+
+      .seats {
+        display: flex;
+      }
+
+      .team-member {
+        cursor: pointer;
+      }
+    </style>
+
+    <lancie-ajax auto-fire id="ajaxGetAllSeats" refurl="seats" on-lancie-ajax="onGetAllSeats"></lancie-ajax>
+    <lancie-ajax auto-fire id="ajaxGetTeamTickets" refurl="tickets/teammembers" on-lancie-ajax="onGetTeamTickets"></lancie-ajax>
+    <lancie-ajax id="ajaxReserveSeat" method="POST" on-lancie-ajax="onReserveSeat"></lancie-ajax>
+    <lancie-ajax id="ajaxAddSeatRow" method="POST" body="[[getSeatGroupDTOBody(seatGroup,seatAmount)]]" on-lancie-ajax="onAddSeatRow" refurl = "seats"></lancie-ajax>
+    <lancie-ajax id="ajaxRemoveSeats" method="DELETE" body="[[getSeatGroupDTOBody(removeGroup,removeAmount)]]" on-lancie-ajax="onRemoveSeats" refurl = "seats"></lancie-ajax>
+    <lancie-ajax id="ajaxLockSeat" method="POST" body="[[lockBoolean]]" on-lancie-ajax="onLockSeat"></lancie-ajax>
+    <lancie-ajax id="ajaxLockGroup" method="POST" body="[[lockBoolean]]" on-lancie-ajax="onLockGroup"></lancie-ajax>
+
+    <paper-input label="Search user" value="{{filterUser}}"></paper-input>
```

Comment:
Not sure we need a search here, I'd vouch for removing it, this just duplicates features from the frontend which we don't need here.